problems/lib/problem1.rb

```diff
@@ -0,0 +1,5 @@
+class Problem1

```

Comment:
Two alternatives would do the job:
1. Monkey patching of the Array class, though I don't like it and definitely wouldn't use it in this case, since I lose the default sum method:
`def Array
  def sum(array)
    array.select(&:even?).sum
  end
end`

2. Add a module with a module level method.